src/common/ceph_time.cc

```diff
@@ -95,7 +95,8 @@ namespace ceph {
 	   typename std::enable_if<Clock::is_steady>::type*>
   std::ostream& operator<<(std::ostream& m,
 			   const std::chrono::time_point<Clock>& t) {
-    return m << std::chrono::duration<double>(t.time_since_epoch()).count()
+    return m << std::fixed << std::chrono::duration<double>(
+		t.time_since_epoch()).count()
```

Comment:
This doesn't make sense to me.  First, why output a time point as a string of seconds since the epoch, and second, why have the 's' suffix?  If the user wants seconds, they should do this cast in the caller to get a double (and, e.g., dump as a json float); otherwise, they should get UTC or something.

IIRC this operator<< for timepoint is almost never used..

Reply:
@adamemerson

Reply:
In context I only intended it to help with debugging prints. In general we shouldn't be printing monotonic time points in any other place, we should be printing the difference of two monotonic time points.

Reply:
oh i see, for monotonic this seems fine

Reply:
oh i see, for monotonic this seems fine